src/styles/_thermostat.scss

```diff
@@ -1,6 +1,234 @@
 @use './colors';
 
-@mixin all {
+@mixin temperature-control-thumb(
```

Comment:
🐠 Do we want to add any stylistic differences between internal and exported mixins?

Reply:
Cc @mikewuu

Reply:
As in class/mixin names? I don't have any strong opinions here. 

Seeing as customizations at the style/component level aren't officially supported, I think we can just aim for consistency.